src/internal/utils.ts

```diff
@@ -14,34 +14,31 @@
  * limitations under the License.
  */
 
-import { AirInterpreter, LogLevel as AvmLogLevel } from '@fluencelabs/avm';
 import log from 'loglevel';
 import { CallServiceData, CallServiceResult, CallServiceResultType, ResultCodes } from './commonTypes';
-import { AvmLoglevel, FluencePeer } from './FluencePeer';
+import { FluencePeer } from './FluencePeer';
 import { Particle, ParticleExecutionStage } from './Particle';
-
-export const createInterpreter = (logLevel: AvmLoglevel): Promise<AirInterpreter> => {
-    const logFn = (level: AvmLogLevel, msg: string) => {
-        switch (level) {
-            case 'error':
-                log.error(msg);
-                break;
-
-            case 'warn':
-                log.warn(msg);
-                break;
-
-            case 'info':
-                log.info(msg);
-                break;
-
-            case 'debug':
-            case 'trace':
-                log.log(msg);
-                break;
-        }
-    };
-    return AirInterpreter.create(logLevel, logFn);
+import { LogLevel as AvmLoglevel } from '@fluencelabs/avm-runner-interface';
+
+export const avmLogFunction = (level: AvmLoglevel, msg: string) => {
```

Comment:
You have `logTo` in `Particle.ts`, they seem very similar. Maybe unify them? 

Cuz it seems that `Particle's` `logTo` still uses `console.trace`, and I remember we decided to avoid it due to it being way too verbose

Reply:
LogFunction moved to AvmRunner